Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.yml

```diff
@@ -9,13 +9,15 @@ configuration:
   required: true
   type: 0
 - name: api_key_id
-  display: API key ID
+  type: 9
```

Comment:
Good job prioritizing this, but please check with @mmhw on how to do this.
This change isn't as simple as just switching the param type.